docs/diagrams/alias/AliasExecutionActivityDiagram.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
```

Comment:
Would it be helpful to future readers to show that the `isRecursiveKeyword` and `isReservedKeyword` check is also done in this process?

Reply:
Will do in later iteration